### PR TITLE
Update the OpenCV-JavaScript introduction page

### LINKED ISSUE #44: Update Introduction page in tutorials
We need to introduce more  of OpenCV-JavaScript and OpenCV-JavaScript Tutorials,  just like OpenCV-Python.

### PR DESCRIPTION
**Pull request by huningxin**
Fix #44

### CODE REVIEW

**Comment by huningxin** [comment]
@pancx , please take a look. Thanks!

**Comment by pancx** [comment]
Just one thing, Shanghai Jiaotong University should be Shanghai Jiao Tong University

File: doc/js_tutorials/js_setup/js_intro/js_intro.markdown [hunk 1]
```diff
@@ -11,15 +11,43 @@ development continued under the support of Willow Garage with Gary Bradsky and V
...
+
+Prior knowledge of JavaScript and web application development is recommended as they won't be covered in this guide.
+
+Contributors
+------------
+
+Below is the list of contributors of OpenCV-JavaScript bindings and tutorials.
+
+-#  Sajjad Taheri (Google Summer of Code 2017 mentor, University of California, Irvine)
+-#  Congxiang Pan (Google Summer of Code 2017 intern, Shanghai Jiaotong University)
```
Comment (pancx) [inline doc/js_tutorials/js_setup/js_intro/js_intro.markdown]: Shanghai Jiaotong University should be Shanghai Jiao Tong University

File: doc/js_tutorials/js_setup/js_intro/js_intro.markdown [hunk 2]
```diff
@@ -11,15 +11,43 @@ development continued under the support of Willow Garage with Gary Bradsky and V
...
+-#  Wenyao Gan (Shanghai Jiaotong University)
```
Comment (pancx) [inline doc/js_tutorials/js_setup/js_intro/js_intro.markdown]: same as above

**Comment by huningxin** [comment]
Thanks for review, @pancx . Updated the commit, please take another look.

**Comment by pancx** [comment]
Looks good！

**Pull request merged**